docs/source/advanced/ipu.rst

```diff
@@ -106,7 +108,7 @@ Note that by default we return the last device iteration loss. You can override
     )
     trainer.fit(model)
 
-You can also override all options by passing the ``poptorch.Options`` to the plugin. See `poptorch options documentation <https://docs.graphcore.ai/projects/poptorch-user-guide/en/latest/batching.html>`_ for more information.
+You can also override all options by passing the ``poptorch.Options`` to the plugin. See `PopTorch options documentation <https://docs.graphcore.ai/projects/poptorch-user-guide/en/latest/batching.html>`__ for more information.
```

Comment:
what does the double underscore do?

Reply:
It generates an "anonymous" link; i.e. without an anchor that can be used to reference it. In general, it often doesn't matter much which is used. But I noticed there was a mixture of styles, so I couldn't help but make them consistent